### ISSUE TITLE
Stripe connect?

### ISSUE THREAD

**Issue by jofftiquez** [vue-stripe/vue-stripe#161]
Check stripe connect docs and see if there's something we can add from there.

**Comment by ramsrib** [comment]
It would be awesome to have stripe connect support here.

**Comment by jofftiquez** [comment]
I agree @ramsrib hopefully there's something we can implement in vue-stripe. So far I have no significant progress yet but I am working on it.

**Comment by samstaron** [comment]
Stripe connect documentation instructs to pass the connected account ID when initialising Stripe. How can I implement with Vue-Stripe? I have a component as per the [Vue-Stripe documentation](https://vuestripe.com/stripe-elements/card) and am already passing the 'pk' prop, but cannot add the connected stripeAccount.

```
// Initialize Stripe.js with the same connected account ID used when creating
// the PaymentIntent.
var stripe = Stripe('pk_test_xxxxxxxxxxxxxxxxx.....', {
  stripeAccount: '{{CONNECTED_STRIPE_ACCOUNT_ID}}'
});
```

**Comment by sebaqe** [comment]
I am also looking at Using this with Stripe Connect for a marketplace. is this being worked on? if not, can I help somehow? 
thanks,
Sebastian

**Comment by davidlapham** [comment]
Any updates on this? It appears the docs say it's supported, however, I can see that the connect_account id is not being passed in the query.

**Comment by jofftiquez** [comment]
Hi everyone. I am contemplating about this and I can't really think of a way to utilize VueStripe for Strip Connect given the nature of the integration of Stripe Connect to the frontend. I might be wrong but I really can't see anything for now. 

As for @samstaron's concern, those options have always been available to the Card elements props. As you can see here:

https://github.com/vue-stripe/vue-stripe/blob/ab6afa7da3529b7d4bc3bb86d849e91a02a0d184/src/elements/Card.vue#L39-L50

https://github.com/vue-stripe/vue-stripe/blob/ab6afa7da3529b7d4bc3bb86d849e91a02a0d184/src/elements/Card.vue#L106-L110

https://github.com/vue-stripe/vue-stripe/blob/ab6afa7da3529b7d4bc3bb86d849e91a02a0d184/src/elements/Card.vue#L121

Also if you want to utilize Stripe.js using VueStripe you can do so by using the [VueStripe Plugin](https://docs.vuestripe.com/vue-stripe/vue-stripe-plugin) so can have access to all Stripe objects directly in the vue context via `this.$stripe` prototype object. The options for this are also already available here:

https://github.com/vue-stripe/vue-stripe/blob/ab6afa7da3529b7d4bc3bb86d849e91a02a0d184/src/stripe/index.js#L11-L13

I feel like we now need to close this one. What do you guys think?
